Documentation/ceph-upgrade.md

```diff
@@ -286,13 +331,29 @@ This cluster is finished:
   rook-version=v1.2.0
 ```
 
-### 4. Verify the updated cluster
+### 5. Verify the updated cluster
 
 At this point, your Rook operator should be running version `rook/ceph:v1.2.0`.
 
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
 
-### 5. Update Rook-Ceph custom resource definitions
+### 6. CSI Manual Update (optional)
+
+If you determined in step 1 that you were affected by the CSI driver restart issue that disconnects
+the application pods from their mounts, continue with this section. Otherwise, you can skip to step 7.
+
+Your cluster should now be in a state where Rook has upgraded everything except the CSI driver.
+The CSI driver pods will not be updated until you delete them manually. This allows you to control
+when your application pods will be affected by the CSI driver restart.
+
+For each node:
+- Drain your application pods from the node
+- Delete the CSI driver pods on the node
+  - The pods to delete will be named with a `csi-cephfsplugin` or `csi-rbdplugin` prefix and have a random suffix on each node.
```

Comment:
Provisioner pods also start with same name do we need to add a caution not to delete Provisioner pods?
```
csi-cephfsplugin-9c7kf                          0/3     ContainerCreating   0          5s
csi-cephfsplugin-p5qq9                          0/3     ContainerCreating   0          5s
csi-cephfsplugin-provisioner-6c7d6f4964-hjdpl   0/4     ContainerCreating   0          5s
csi-cephfsplugin-provisioner-6c7d6f4964-m4lvj   0/4     ContainerCreating   0          5s
csi-rbdplugin-jpwq6                             0/3     ContainerCreating   0          6s
csi-rbdplugin-provisioner-54cc7d5848-bbcnc      0/5     ContainerCreating   0          6s
csi-rbdplugin-provisioner-54cc7d5848-jx7rn      0/5     ContainerCreating   0          6s
csi-rbdplugin-xcrpr                             0/3     ContainerCreating   0          6s
```

Reply:
Yes, we should clarify thanks